WalletWasabi/Crypto/StrobeProtocol/StrobeHasher.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using NBitcoin;
+using WalletWasabi.WabiSabi;
+
+namespace WalletWasabi.Crypto.StrobeProtocol
+{
+	public sealed class StrobeHasher
```

Comment:
Add some description or link here, what is this doing.

Reply:
This class is absurdly easy to understand, it simple calculates hashes using our `Strobe128` implementation and that class contains the links to the Merlin framework page, the Strobe protocol white paper and the rust implementation we ported. I think it makes no sense to comment anything here other than "See: Strobe128 class"

Reply:
ACK